app/assets/stylesheets/common/components/notifications-tracking.scss

```diff
@@ -37,3 +37,7 @@
     text-align: left;
   }
 }
+
+.notifications-tracking-content {
```

Comment:
Should this be scoped so that it only applies when in the "nested dmenu" mobile view? If we apply it to desktop, then I guess it's going to make the tracking dropdown appear on top of the header?

Right now it looks like this on desktop:

<img width="677" alt="SCR-20250331-kkhi" src="https://github.com/user-attachments/assets/35b26168-e3ea-4423-bf54-de44e2e4ba61" />

Reply:
I don't think it's scoped to mobile view, but yes maybe only for the case in timeline